IGNORE.py

```diff
@@ -0,0 +1,87 @@
+import turtle, random
+
+def draw_sunflower(t):
+    for _ in range(36):
+        t.circle(50)
+        t.right(10)
+
+def draw_rose(t):
+    for _ in range(5):
+        t.circle(60, 180)
+        t.left(144)
+
+def draw_daisy(t):
+    for _ in range(12):
+        t.circle(40)
+        t.left(30)
+
+def draw_tulip(t):
+    t.begin_fill()
+    for _ in range(2):
+        t.circle(40, 90)
+        t.circle(40 // 2, 90)
+    t.end_fill()
+
+def draw_lily(t):
+    for _ in range(5):
+        t.circle(100, 60)
+        t.left(144)
```

Comment:
@RohitDen In the selected code what is the _ doing?

Reply:
The _ in the selected code is used to be a variable name in a for loop so that a lily is drawn 5 times